index.js

```diff
@@ -0,0 +1,2 @@
+const createGrid = Array.from({ length: 3 }, () => [undefined, undefined, undefined]);
+const displayGrid = () => console.log(createGrid());
```

Comment:
This really should have an empty line at the end of files

```suggestion
const displayGrid = () => console.log(createGrid());

```